RiotSwiftUI/Modules/Room/PollTimeline/View/PollTimelineAnswerOptionButton.swift

```diff
@@ -0,0 +1,155 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct PollTimelineAnswerOptionButton: View {
+    
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    @Environment(\.theme) private var theme: ThemeSwiftUI
+    
+    let answerOption: TimelineAnswerOption
+    let pollClosed: Bool
+    let showResults: Bool
+    let totalAnswerCount: UInt
+    let action: () -> Void
+    
+    // MARK: Public
+    
+    var body: some View {
+        Button(action: action) {
+            let rect = RoundedRectangle(cornerRadius: 4.0)
+            answerOptionLabel
+                .padding(.horizontal, 8.0)
+                .padding(.top, 12.0)
+                .padding(.bottom, 4.0)
+                .clipShape(rect)
+                .overlay(rect.stroke(borderAccentColor, lineWidth: 1.0))
+                .accentColor(progressViewAccentColor)
+        }
+    }
+    
+    var answerOptionLabel: some View {
+        VStack(alignment: .leading, spacing: 12.0) {
+            HStack(alignment: .top, spacing: 8.0) {
+                
+                if !pollClosed {
+                    Image(uiImage: answerOption.selected ? Asset.Images.pollCheckboxSelected.image : Asset.Images.pollCheckboxDefault.image)
+                }
+                
+                Text(answerOption.text)
+                    .font(theme.fonts.body)
+                    .foregroundColor(theme.colors.primaryContent)
+                
+                if pollClosed && answerOption.winner {
+                    Spacer()
+                    Image(uiImage: Asset.Images.pollWinnerIcon.image)
+                }
+            }
+            
+            HStack {
+                ProgressView(value: Double(showResults ? answerOption.count : 0),
+                             total: Double(totalAnswerCount))
+                    .progressViewStyle(LinearProgressViewStyle())
+                    .scaleEffect(x: 1.0, y: 1.2, anchor: .center)
+                    .padding([.top, .bottom], 8.0)
```

Comment:
You can also use `.vertical`. Other than that, this looks much better to me 🙂
```suggestion
                    .padding(.vertical, 8.0)
```